src/devicediscoveryextension.cs

```diff
@@ -0,0 +1,104 @@
+//
+// DeviceDiscoveryExtension C# bindings
+//
+// Authors:
+//	Israel Soto  <issoto@microsoft.com>
+//
+// Copyright 2022 Microsoft Corporation.
+//
+
+using System;
+using ObjCRuntime;
+using Foundation;
+using UniformTypeIdentifiers;
+
+using nw_endpoint_t=System.IntPtr;
```

Comment:
This isn't a standard type name in C#.

Reply:
but it does exists in the SDK under `src/Network/NWEndpoint.cs`

Reply:
Marked the property using this alias as `[Internal]` and exposed the property with the correct C# type.